reference.md

```diff
@@ -9,35 +9,46 @@ title: Knowledge Base
-* [Translating between Slurm and PBS](https://www.msi.umn.edu/slurm/pbs-conversion) from University of Minnesota
+* [Torque/PBS summary](
+  https://gif.biotech.iastate.edu/torque-pbs-job-management-cheat-sheet)
+  from Iowa State
+* [Translating between Slurm and PBS](
+  https://www.msi.umn.edu/slurm/pbs-conversion) from University of Minnesota
 
 ### Units and Language
 
-A computer's memory and disk are measured in units called *Bytes* (one Byte is 8 bits). As today's
-files and memory have grown to be large given historic standards, volumes are noted using the
-[SI](https://en.wikipedia.org/wiki/International_System_of_Units) prefixes. So 1000 Bytes is a
-Kilobyte (kB), 1000 Kilobytes is a Megabyte (MB), 1000 Megabytes is a Gigabyte (GB), etc.
+A computer's memory and disk are measured in units called *Bytes* (one Byte is
+8 bits). As today's files and memory have grown to be large given historic
+standards, volumes are noted using the
+[SI](https://en.wikipedia.org/wiki/International_System_of_Units) prefixes. So
+1000 Bytes is a Kilobyte (kB), 1000 Kilobytes is a Megabyte (MB), 1000
+Megabytes is a Gigabyte (GB), etc.
 
-History and common language have however mixed this notation with a different meaning. When people
-say "Kilobyte", they mean 1024 Bytes instead. In that spirit, a Megabyte is 1024 Kilobytes.
+History and common language have however mixed this notation with a different
+meaning. When people say "Kilobyte", they mean 1024 Bytes instead. In that
+spirit, a Megabyte is 1024 Kilobytes.
 
 To address this ambiguity, the [International System of Quantities](
-https://en.wikipedia.org/wiki/International_System_of_Quantities) standardizes the *binary* prefixes
-(with base of 2<sup>10</sup>=1024) by the prefixes Kibi (ki), Mibi (Mi), Gibi (Gi), etc. For more
-details, see [here](https://en.wikipedia.org/wiki/Binary_prefix).
+https://en.wikipedia.org/wiki/International_System_of_Quantities) standardizes
+the *binary* prefixes (with base of 2<sup>10</sup>=1024) by the prefixes Kibi
+(ki), Mibi (Mi), Gibi (Gi), etc. For more details, see
+[here](https://en.wikipedia.org/wiki/Binary_prefix).
 
 ### "No such file or directory" or "symbol 0096" errors
 
-`scp` and `rsync` may throw a perplexing error about files that very much do exist. One source of
-these errors is copy-and-paste of command line arguments from Web browsers, where the double-dash
-string `--` is rendered as an em-dash character `—` (or en-dash `–`, or horizontal bar `―`). For
-example, instead of showing the transfer rate in real time, the following command fails mysteriously.
+`scp` and `rsync` may throw a perplexing error about files that very much do
+exist. One source of these errors is copy-and-paste of command line arguments
+from Web browsers, where the double-dash string `--` is rendered as an em-dash
+character `—` (or en-dash `–`, or horizontal bar `―`). For example, instead of
```

Comment:
Do we want the HTML-ized versions of these characters that we typically use elsewhere, or the unicode versions that we have here?

Reply:
cdbf42e